src/v0/destinations/klaviyo/transform.js

```diff
@@ -156,7 +157,8 @@ const trackRequestHandler = (message, category, destination) => {
   const payload = {};
   const { privateApiKey, flattenProperties } = destination.Config;
   let event = get(message, 'event');
-  event = event ? event.trim().toLowerCase() : event;
```

Comment:
@anantjain45823  Were we allowing track events without name before ?
If yes, this would be breaking change.
Check logic here https://github.com/rudderlabs/rudder-transformer/pull/2992/files#diff-7bf116831bc60affad855f69064fa73f991645698a1ea61b7e6f12badfc9fe1eL212 where category.url is being picked up for events without name

Reply:
Yes we were passing it for undefined as well I have changed it and added a test for `undefined` case as well